tests/test_result/test_result_base.py

```diff
@@ -1,7 +1,7 @@
-from returns.result import Result
+from returns.result import Failure, Success
```

Comment:
This test can be deleted, the only purpose was to ensure we put a type inside `Result.success_type` and `Result.failure_type`